mesh/profiles/views.py

```diff
@@ -21,26 +21,81 @@ def bio_view(request):
 
 def profile_picture(request):
     if request.method == "GET":
-        data = request.GET
         response = {
             "status": "success"
         }
-        if data.get("accountID") is None:
-            response.update({"status": "error"})
-            response.update({"message": "Missing account ID."})
-            return JsonResponse(response)
-        else:
-            try:
-                profile = Profile.objects.get(accountID=int(data.get("accountID")))
-                response.update({
-                    "data": {
-                        "get": {
-                            "profilePicture": profile.image.url
-                        }
+        profile = get_profile(request.GET, response)
+        if profile is not None:
+            response.update({
+                "data": {
+                    "get": {
+                        "profilePicture": profile.image.url
+                    }
+                }
+            })
```

Comment:
There are no error handlings. What if the profile is none? Right now it will return a success.